plugins/notion/src/MapFields.tsx

```diff
@@ -1,67 +1,65 @@
 import { GetDatabaseResponse } from "@notionhq/client/build/src/api-endpoints"
-import { CollectionField } from "framer-plugin"
-import { assert, isDefined } from "./utils"
+import { ManagedCollectionField } from "framer-plugin"
+import { assert } from "./utils"
 import {
     NotionProperty,
     PluginContext,
     SynchronizeMutationOptions,
+    SynchronizeProgress,
     getCollectionFieldForProperty,
     getPossibleSlugFields,
-    hasFieldConfigurationChanged,
-    pageContentField,
+    hasDatabaseFieldsChanged,
+    isSupportedNotionProperty,
+    supportedCMSTypeByNotionPropertyType,
     richTextToPlainText,
+    getNotionProperties,
+    hasFieldConfigurationChanged,
 } from "./notion"
-import { useMemo, useState } from "react"
+import { Fragment, useMemo, useState } from "react"
 import classNames from "classnames"
 import { IconChevron } from "./components/Icons"
 import { Button } from "./components/Button"
 import { isFullDatabase } from "@notionhq/client"
 import { CheckboxTextfield } from "./components/CheckboxTexfield"
 
-interface CollectionFieldConfig {
-    field: CollectionField | null
-    isNewField: boolean
-    originalFieldName: string
-}
+function getSortedProperties(database: GetDatabaseResponse): NotionProperty[] {
+    return getNotionProperties(database).sort((propertyA, propertyB) => {
+        // Properties that are not supported in the Plugin are displayed at the bottom of the list.
+        if (isSupportedNotionProperty(propertyA) && isSupportedNotionProperty(propertyB)) {
+            return -1
+        } else if (isSupportedNotionProperty(propertyA)) {
+            return -1
+        } else if (!isSupportedNotionProperty(propertyB)) {
+            return 1
+        }
```

Comment:
I might have flipped the logic here but isn't this the same?
```ts
const a = isSupportedNotionProperty(propertyA) ? 1 : 0
const b = isSupportedNotionProperty(propertyB) ? 1 : 0
return a - b
```

Reply:
Yep, works for me 👌